core/src/com/unciv/logic/civilization/managers/TurnManager.kt

```diff
@@ -217,6 +217,10 @@ class TurnManager(val civInfo: Civilization) {
 
         civInfo.notifications.clear()
 
+        if (civInfo.isMajorCiv() && civInfo.isAlive()) {
+            civInfo.statsHistory.recordRankingStats(civInfo)
+        }
```

Comment:
That's a tricky one. I was thinking about whether beginning of turn or end of turn represents the "score at turn" concept better. But yes, this would not record the turn at which a cilization actually gets defeated, but the last entry would be one turn before. Let me just move this to the start of the turn then?